app/common/AreaFilterSelect.jsx

```diff
@@ -42,18 +42,40 @@ class AreaFilterSelect extends React.Component {
       return null;
     };
 
-    return (
-      <div className="area-filter-select">
-        <Select
-          name="area-filter-select"
-          placeholder="Filter by Administrative Boundary"
-          value={this.state.selectedLayer}
-          options={this.props.options}
-          onChange={this.handleSelectChange}
-        />
-        { polygonSelectorComponent() }
-      </div>
-    );
+    const {
+      exploreANeighborhoodSelected,
+      ntaSelection,
+    } = this.props;
+
+    if (exploreANeighborhoodSelected) {
+      this.state.selectedLayer = { value: 'nta2020', label: 'Neighborhood Tabulation Area' };
```

Comment:
Use `this.setState()` please.

Reply:
Hmm, Apparently `setState()`  should not be used within a render.

```
Warning: setState(...): Cannot update during an existing state transition (such as within `render` or another component's constructor). Render methods should be a pure function of props and state; constructor side-effects are an anti-pattern, but can be moved to `componentWillMount`.
```